plugin/trino-prometheus/src/test/java/io/trino/plugin/prometheus/TestPrometheusIntegration.java

```diff
@@ -15,39 +15,53 @@
 
 import com.google.common.collect.ImmutableMap;
 import io.airlift.log.Logger;
+import io.airlift.units.Duration;
+import io.trino.spi.connector.ConnectorSplitSource;
+import io.trino.spi.connector.DynamicFilter;
 import io.trino.testing.AbstractTestQueryFramework;
 import io.trino.testing.MaterializedResult;
 import io.trino.testing.MaterializedRow;
 import io.trino.testing.QueryRunner;
 import okhttp3.HttpUrl;
 import okhttp3.OkHttpClient;
 import okhttp3.Request;
+import org.testng.annotations.AfterClass;
 import org.testng.annotations.Test;
 
 import java.util.concurrent.TimeUnit;
 
+import static io.trino.plugin.prometheus.PrometheusQueryRunner.createPrometheusClient;
 import static io.trino.plugin.prometheus.PrometheusQueryRunner.createPrometheusQueryRunner;
+import static io.trino.spi.connector.NotPartitionedPartitionHandle.NOT_PARTITIONED;
+import static java.util.concurrent.TimeUnit.DAYS;
+import static java.util.concurrent.TimeUnit.SECONDS;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.fail;
 
-/**
- * Integration tests against Prometheus container
- */
-@Test(singleThreaded = true)
-public class TestPrometheusIntegrationStatus
+public class TestPrometheusIntegration
         extends AbstractTestQueryFramework
 {
+    private static final int NUMBER_MORE_THAN_EXPECTED_NUMBER_SPLITS = 100;
+
     private PrometheusServer server;
+    private PrometheusClient client;
 
     @Override
     protected QueryRunner createQueryRunner()
             throws Exception
     {
         this.server = new PrometheusServer();
```

Comment:
Thus, you don't need `destroy()` method. 
```suggestion
        this.server = closeAfterClass(new PrometheusServer());
```

Reply:
Thanks for the comments! Changed.